examples/outsim-advanced-ts/src/index.ts

```diff
@@ -0,0 +1,45 @@
+import './env.ts';
+
+import debug from 'debug';
+import { OutSim, OutSimPack2 } from 'node-insim';
```

Comment:
`OutSimPack2` should be re-exported in `src/out/index.ts` and `src/index.ts` so it becomes available here.

Reply:
re-exported